core/arch/arm/kernel/tz_ssvce_pl310_a32.S

```diff
@@ -96,7 +96,7 @@ loop_cli_sync:
 	cmp	r1, #0
 	bne	loop_cli_sync
```

Comment:
Not strictly related to the change. Is this loop needed?

Reply:
If only PL310 is supported, it loop is not needed. L220 needs that loop :) Keep that loop for PL310 seems no harm.

Reply:
Thanks for the explanation. A comment in the code would be welcome. :-)